libretroshare/src/gxs/rsdataservice.cc

```diff
@@ -1340,15 +1337,16 @@ void RsDataService::locked_retrieveGrpMetaList(RetroCursor *c, std::map<RsGxsGro
 
 	while(valid)
 	{
-		RsGxsGrpMetaData* m = locked_getGrpMeta(*c, 0,true);
+        auto m = locked_getGrpMeta(*c, 0,true);
 
-		if(m)
+        if(m != nullptr)

```

Comment:
You probably alrrady know that the `if(m)` syntax works also with `shared_ptr<T>` ;)